applications/StructuralMechanicsApplication/tests/shell_test/Shell_Q4_Thin_linear_dynamic_test_materials.json

```diff
@@ -0,0 +1,19 @@
+{
+    "properties": [{
+            "model_part_name": "Parts_surface",
+            "properties_id": 1,
+            "Material": {
+                    "name": "Material",
+                    "constitutive_law": {
+                            "name": "KratosMultiphysics.StructuralMechanicsApplication.LinearElasticPlaneStress2DLaw"
+                    },
+                    "Variables": {
```

Comment:
With the recent changes it is better not to put the appname before the variable in the json